File: Example/CedarUnitTests/mocks/StringCell.m
//
//  StringCell.m
//  DTTableViewManager
//
//  Created by Denys Telezhkin on 11.02.14.
//  Copyright (c) 2014 Denys Telezhkin. All rights reserved.
//

#import "StringCell.h"
@interface StringCell()
@property (nonatomic, retain) NSString * string;
@end

@implementation StringCell

-(void)updateWithModel:(id)model
{
    self.textLabel.text = model;
    self.string = model;
}

-(id)model
{
    return self.string;
}

@end

File: Example/Example/Cells/StringCell.swift
//
//  StringCell.swift
//  DTTableViewManager
//
//  Created by Denys Telezhkin on 02.08.15.
//  Copyright (c) 2015 Denys Telezhkin. All rights reserved.
//

import UIKit
import ModelStorage
import DTTableViewManager

class StringCell: UITableViewCell, ModelTransfer {
    func updateWithModel(model: String) {
        self.textLabel?.text = model
    }

}
